Source/Core/DolphinQt2/ToolBar.cpp

```diff
@@ -91,15 +92,13 @@ void ToolBar::MakeActions()
 
 void ToolBar::UpdateIcons()
 {
-  QString dir = Settings().GetThemeDir();
-
-  m_open_action->setIcon(QIcon(QStringLiteral("open.png").prepend(dir)));
-  m_paths_action->setIcon(QIcon(QStringLiteral("browse.png").prepend(dir)));
-  m_play_action->setIcon(QIcon(QStringLiteral("play.png").prepend(dir)));
-  m_pause_action->setIcon(QIcon(QStringLiteral("pause.png").prepend(dir)));
-  m_stop_action->setIcon(QIcon(QStringLiteral("stop.png").prepend(dir)));
-  m_fullscreen_action->setIcon(QIcon(QStringLiteral("fullscreen.png").prepend(dir)));
-  m_screenshot_action->setIcon(QIcon(QStringLiteral("screenshot.png").prepend(dir)));
-  m_config_action->setIcon(QIcon(QStringLiteral("config.png").prepend(dir)));
-  m_controllers_action->setIcon(QIcon(QStringLiteral("classic.png").prepend(dir)));
+  m_open_action->setIcon(QIcon(Resources::GetScaledThemeIcon(QStringLiteral("open"))));
+  m_paths_action->setIcon(QIcon(Resources::GetScaledThemeIcon(QStringLiteral("browse"))));
+  m_play_action->setIcon(QIcon(Resources::GetScaledThemeIcon(QStringLiteral("play"))));
+  m_pause_action->setIcon(QIcon(Resources::GetScaledThemeIcon(QStringLiteral("pause"))));
+  m_stop_action->setIcon(QIcon(Resources::GetScaledThemeIcon(QStringLiteral("stop"))));
+  m_fullscreen_action->setIcon(QIcon(Resources::GetScaledThemeIcon(QStringLiteral("fullscreen"))));
+  m_screenshot_action->setIcon(QIcon(Resources::GetScaledThemeIcon(QStringLiteral("screenshot"))));
+  m_config_action->setIcon(QIcon(Resources::GetScaledThemeIcon(QStringLiteral("config"))));
+  m_controllers_action->setIcon(QIcon(Resources::GetScaledThemeIcon(QStringLiteral("classic"))));
```

Comment:
Why not have `GetScaledThemeIcon` return a QIcon directly? It has "Icon" in the name already.